app/src/main/java/org/oppia/android/app/player/state/StateFragmentPresenter.kt

```diff
@@ -449,7 +451,7 @@ class StateFragmentPresenter @Inject constructor(
     val inputManager: InputMethodManager =
       activity.getSystemService(Context.INPUT_METHOD_SERVICE) as InputMethodManager
     inputManager.hideSoftInputFromWindow(
-      fragment.view!!.windowToken,
+      fragment.view?.windowToken,
```

Comment:
Is this change still needed now that replace() is being used?

Reply:
Even after using .replace still it's retuning null and we need to use ? to handle that.

Reply:
Here, keeping this null check to avoid app crashing on null view.